aws/resource_aws_codebuild_project.go

```diff
@@ -324,6 +358,29 @@ func expandProjectEnvironment(d *schema.ResourceData) *codebuild.ProjectEnvironm
 	return projectEnv
 }
 
+func expandVpcConfig(d *schema.ResourceData) *codebuild.VpcConfig {
+	configs := d.Get("vpc_config").(*schema.Set).List()
+	data := configs[0].(map[string]interface{})
```

Comment:
This line is generating a panic for me running the acceptance test:
```
=== RUN   TestAccAWSCodeBuildProject_basic
panic: runtime error: index out of range

goroutine 2579 [running]:
github.com/terraform-providers/terraform-provider-aws/aws.expandVpcConfig(0xc42084b110, 0x3715e47)
	/Users/bflad/go/src/github.com/terraform-providers/terraform-provider-aws/aws/resource_aws_codebuild_project.go:363 +0x790
```

It will go away with the function signature refactor and TypeList changes mentioned above.